### PR TITLE
[feature] [3] Initial implementation

### LINKED ISSUE #3: Add transliteration page
Typing Ukrainian text in the upper text field, the user receives the converted text in the lower field.

image

### PR DESCRIPTION
**Pull request by neodavis**

### CODE REVIEW

**Review by Pagendera — APPROVED** [review]
Nice

File: transliteration-page/transliteration-page.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    output.value = transliteratedString;
+}
+
+function getTransliteratedString(sourceString, transliterateMap) {
+    return [...sourceString]
+        .map(sourceLetter => transliterateMap[sourceLetter] || sourceLetter)
+        .join('');
+}
+
+handleSourceChange(source, result, ukrainianToEnglishTransliterationMap);
```
Comment (ashotvdovychenko) [inline transliteration-page/transliteration-page.js:92]: I don't really understand the purpose of second `handleSourceChange()` method without body.

File: transliteration-page/transliteration-page.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+const source = document.querySelector('#area-source');
+const result = document.querySelector('#area-result');
```
Comment (ashotvdovychenko) [inline transliteration-page/transliteration-page.js:2]: Where these constans are used?

**Review by ashotvdovychenko — APPROVED** [review]

File: transliteration-page/transliteration-page.js (hunk 1 above)
Comment (neodavis) [reply transliteration-page/transliteration-page.js:92]: This is just a regular call of `handleSourceChange()` with passing
params `source`, `result` and `ukrainianToEnglishTransliterationMap`.

File: transliteration-page/transliteration-page.js (hunk 2 above)
Comment (neodavis) [reply transliteration-page/transliteration-page.js:2]: 92 row, call of the `handleSourceChange()` function

**Pull request merged**